File: server/notification-service/src/main/java/com/sudoo/notification/controller/dto/SupplierInfoDto.java
package com.sudoo.notification.controller.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SupplierInfoDto {
    private String supplierId;
    private String userId;
    private String name;
    private String brand;
    private String locationName;
    private String contactUrl;
    private Float rate;
}

File: server/product-service/src/main/kotlin/com/sudoo/productservice/dto/SupplierInfoDto.kt
package com.sudoo.productservice.dto

data class SupplierInfoDto(
    val supplierId: String? = null,
    val userId: String,
    val name: String,
    val avatar: String,
    val brand: String,
    val locationName: String,
    val contactUrl: String,
    val rate: Float
)